app/helpers/katello/katello_url_helper.rb

```diff
@@ -8,17 +8,24 @@ module KatelloUrlHelper
     end
 
     def kurl_valid?(url)
-      valid_for_prefixes(url, PROTOCOLS)
+      validate_host(url)
```

Comment:
if validate_host(url) returns false, but validate_scheme(url, PROTOCOLS) returns true, this method will still return true because is returning the last line, right?